.github/workflows/devbuild.yml

```diff
@@ -641,3 +641,34 @@ jobs:
         with:
           name: modmesh-pilot-win64
           path: modmesh-pilot-win64/
+
+  send_email_on_failure:
```

Comment:
Is it possible to have some retry machenism while CI fails on master, so it will not be a spam? The idea is we want to receive the mail when the CI is really broken.

Reply:
Sounds good.  Please create an issue to track the research.  We keep the current implementation for now.